model/src/main/java/org/mskcc/cmo/metadb/model/SampleMetadata.java

```diff
@@ -3,140 +3,183 @@
 import com.fasterxml.jackson.annotation.JsonAlias;
```

Comment:
lots of notes regarding fields that we arent getting from sample manifest endpoint since the return type is SampleManifest and not `SampleMetadata` in IGO LIMS which DOES contain these extra fields